src/main/java/nl/nn/testtool/web/ApiAuthorizationFilter.java

```diff
@@ -91,6 +91,7 @@ public void setObserverRoles(List<String> observerRoles) {
 		addConfigurationPart("PUT/"  + ApiServlet.LADYBUG_API_PATH + "/testtool/node-link-strategy$", observerRoles);
 		addConfigurationPart("GET/"  + ApiServlet.LADYBUG_API_PATH + "/metadata/.*$", observerRoles);
 		addConfigurationPart("GET/"  + ApiServlet.LADYBUG_API_PATH + "/report/.*$", observerRoles);
+		addConfigurationPart("POST/" + ApiServlet.LADYBUG_API_PATH + "/collector/.*$", observerRoles);
```

Comment:
I think we need to create a separate role. Can you remove this line and add the following below the setTesterRoles() method?

	public void setWebServiceRoles(List<String> webServiceRoles) {
		if (constructorDone) log.info("Set web service roles");
		addConfigurationPart("POST/" + ApiServlet.LADYBUG_API_PATH + "/collector/.*$", webServiceRoles);
	}

And add setWebServiceRoles(null); below setTesterRoles(null);?

Reply:
Added the new role